app/pkg/email/email.go

```diff
@@ -11,6 +12,14 @@ import (
 
 var cache = make(map[string]*template.Template, 0)
 
+//Auth creates an auth object for sending mail
```

Comment:
If you can just move this function to the `smtp` package we can get this merged. The `email` package is supposed to be a common ground for all different sort of email providers and this method is very specific to SMTP.

Reply:
My golang skills couldn't work out how to do this... Any pointers for reading for understanding what's going on?

Reply:
Every folder is called a package by the name after the last slash. So app/pkg/email is the email package. app/pkg/email/smtp is the smtp package. 

It’s organized in such a way that we can have one package for each email provider (smtp, mailgun, sendgrid, etc), while the email package contains shared structs/methods common for all providers.

So when I say to move things from email package to smtp package, is to basically move the “Auth” function to the app/pkg/email/smtp folder, inside the smtp.go file.

You might have noticed that the Go stdlib has a smtp package already, so that’s why in the beginning of the smtp.go file I imported that package and renamed to gosmtp, so you’ll need to call the stdlib function like  gosmtp.Auth(...)

Did it help? 😀